lib/get-filesizes-per-type.js

```diff
@@ -0,0 +1,25 @@
+const fs = require('fs');
+const path = require('path');
+
+function getFileSizesPerType(files) {
+	const sizes = getFileSizes(files);
+
+	return files.reduce((typeSizes, file) => {
```

Comment:
within this scope you can simplify / shorten your variable names `sizes`, `type`, `size`, `total`. So: `sizes[type] = total + size;` instead of `typeSizes[fileType] = totalForType + fileSize;`.